#27 - Bump PHP to 8.1.2 and Composer to 2.2.5 (#28)
blumilksoftware/php: readme.md

    && echo "xdebug.log_level=${XDEBUG_LOG_LEVEL}" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
;fi
```

> ---
> #### change Composer version
> if you want to use another Composer version you can override it in Dockerfile:
> ```dockerfile

Reviewer: ```suggestion
#### Change Composer version
If you would like to use a different version of the Composer, you can override it in the Dockerfile:
```dockerfile
```
